src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int SIDE_VALUES = 20;
+    private static final int FIGURE_NUM = 5;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int number = random.nextInt(FIGURE_NUM);
+        if (number == 1) {
+            return new Circle(random.nextInt(SIDE_VALUES), colorSupplier.getRandomColor());
+        } else if (number == 2) {
```

Comment:
do we need `else` after `return` in the previous `if` block?

Reply:
No. I will corect it now